x-pack/plugins/session_view/public/components/process_tree_node/index.tsx

```diff
@@ -150,6 +150,8 @@ export function ProcessTreeNode({
     : 'sessionView:processTreeNodeForkIcon';
   const processIcon = hasExec ? 'console' : 'branch';
 
+  const timeStampsNormal = formatDate(process.getDetails().process.start, KIBANA_DATE_FORMAT);
```

Comment:
you should be able to get `process.getDetails().process.start` from the processDetails. You can do `const { user, start } = processDetails;` above